### PR TITLE
PAY-744: Order payment status reports

### PR DESCRIPTION
**Pull request by shrielenee**
## Purpose of this pull request

This pull request (PR) adds a new topic to the Financial reporting section, Order payment status.

Jira issue [PAY-744](https://jira.corp.magento.com/browse/PAY-744)

Preview build [33](https://docs.magedevteam.com/33/user-guide/magento-payments/order-payment-status.html)

## Magento release version

- [ ] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- user-guide/magento-payments/order-payment-status.html

## Links to Magento source code or PRs

- ...
- ...

### CODE REVIEW

**Review by cruxburn — APPROVED** [review]
Structure of doc is good, I don't like the naming of "Order payment status(es)" for the report heading and we should determine final naming closer to EAP

File: src/magento-payments/order-payment-status.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+The Order payment status view shows extensive info for each status shown in the grid.
+
+### Column descriptions
+
+Order payment status reports include the following information.
+
+Column | Description
+------------ | --------------------
+Order ID | Magento order ID<br> <br>Click the ID to see related [order info](https://docs.magento.com/user-guide/sales/orders.html).
```
Comment (leslietilling) [inline src/magento-payments/order-payment-status.md:71]: We use relative links for topics within the user guide and not the HTML URL. 

```suggestion
Order ID | Magento order ID<br> <br>Click the ID to see related [order info]({% link sales/orders.md %})
```

File: src/magento-payments/order-payment-status.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Order Date | Order date timestamp
+Authorized Date | Date timestamp of payment authorization
+Order Status | Current Magento [order status](https://docs.magento.com/user-guide/sales/order-status.html)
```
Comment (leslietilling) [inline src/magento-payments/order-payment-status.md:74]: same here
```suggestion
Order Status | Current Magento [order status]({% link sales/order-status.md %})
```

**Review by leslietilling — APPROVED** [review]
Looks good, just need to replace hardcoded URLs for internal links.

File: src/magento-payments/order-payment-status.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+## Default order payment status timeframes
+
+These order payment status timeframes are currently available in Magento Payments.
+
+Report | Description
+------------ | --------------------
+Yesterday | Available from the Order payment status dates selector, this shows information for the prior date.
+Today | Available from the Order payment status dates selector, this shows information for the current day.
+Last 7 days | Available from the Order payment status dates selector, this shows information for the last seven days.
+Last 30 days | Available from the Order payment status dates selector and by default in the Order paymen statuses view, this shows information for the last 30 days.
```
Comment (leslietilling) [inline src/magento-payments/order-payment-status.md:56]: typo

```suggestion
Last 30 days | Available from the Order payment status dates selector and by default in the Order payment statuses view, this shows information for the last 30 days.
```

**Comment by shrielenee** [comment]
> Structure of doc is good, I don't like the naming of "Order payment status(es)" for the report heading and we should determine final naming closer to EAP

I am bringing this up with Mike to see if he has any ideas.

**Pull request merged**